old.txt

```diff
@@ -0,0 +1,70 @@
+
+
+
+
+=======
+  is_strict : bool;
+  want_meta : bool;
+  want_exn : bool;
+  variants : variants;
+}
+
+let default_options = {
+  is_strict = true;
+  want_meta = false;
+  want_exn = false;
+  variants = `Array;
+}
+
+let parse_options options =
+  let get_bool = Ppx_deriving.Arg.(get_expr ~deriver bool) in
+  options |> List.fold_left (fun options (name, expr) ->
+    match name with
+    | "strict" -> {options with is_strict = get_bool expr}
+    | "meta" -> {options with want_meta = get_bool expr}
+    | "exn" -> {options with want_exn = get_bool expr}
+    | "variants" -> {options with variants = Ppx_deriving.Arg.get_expr ~deriver variants_conv expr}
+    | _ -> raise_errorf ~loc:expr.pexp_loc "%s does not support option %s" deriver name
+  ) default_options
+>>>>>>> b25db84 (Support alternative representations of variants)
```

Comment:
This looks a conflict marker, are you sure you intended to include this in the PR?